packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -249,35 +249,28 @@ export class Create {
                         break;
 
                     case "dirblk":
-                    // Validate non-zero if dsorg is PS
-                        if (tempOptions.dirblk !== 0 && tempOptions.dsorg === "PS") {
+                        // Validate non-zero if dsorg starts with "PS"
+                        if (tempOptions.dirblk !== 0 && tempOptions.dsorg.startsWith("PS")) {
                             throw new ImperativeError({ msg: ZosFilesMessages.invalidPSDsorgDirblkCombination.message });
                         }
-                        // Validate non-zero if 'dsorg' is PO
-                        if (tempOptions.dirblk === 0 && tempOptions.dsorg === "PO") {
+                        // Validate non-zero if dsorg starts with "PO"
+                        if (tempOptions.dirblk === 0 && tempOptions.dsorg.startsWith("PO")) {
                             throw new ImperativeError({ msg: ZosFilesMessages.invalidPODsorgDirblkCombination.message });
                         }
-
                         break;
 
-                    case "dsntype": {
-                    // Key to create a PDSE.
-                        const type: string = tempOptions.dsntype.toUpperCase();
-                        const availableTypes = ["BASIC", "EXTPREF", "EXTREQ", "HFS", "LARGE", "PDS", "LIBRARY", "PIPE"];
```

Comment:
Wouldn't the removal of the test of availableTypes remove all validation of the dsnType command line option? I could supply an arbitrary string (like 'quicksand') as a dsnType. Have we made a conscience decision to let z/OS do the validation and return the z/OS message if an error occurs? Could this be considered a breaking change (admittedly an edge-case breakage)?

Reply:
TL;DR:  yeah, we want this back 🙏 
---

This is my fault. 
I was confused on the extent of our validation for `dsntype` and how that impacted the DS creation process.
Since this list matches the items listed on z/OSMF, I believe there is no need for us to let the request fail on z/OS.

**_Source_**: https://www.ibm.com/docs/en/zos/3.1.0?topic=dp-subparameter-definition-8